src/portable/raspberrypi/rp2040/rp2040_usb.c

```diff
@@ -329,15 +363,15 @@ bool __tusb_irq_path_func(hw_endpoint_xfer_continue)(struct hw_endpoint *ep)
     pico_trace("Completed transfer of %d bytes on ep %d %s\n",
                ep->xferred_len, tu_edpt_number(ep->ep_addr), ep_dir_string[tu_edpt_dir(ep->ep_addr)]);
     // Notify caller we are done so it can notify the tinyusb stack
-    _hw_endpoint_lock_update(ep, -1);
+    hw_endpoint_lock_update(ep, -1);
     return true;
   }
   else
   {
-    _hw_endpoint_start_next_buffer(ep);
+    hw_endpoint_start_next_buffer(ep);
```

Comment:
rp2040_critical_frame_period() should also be checked here as well, since this will also trigger an usb transaction. I think this place will likely to cause the issue than one in dcd_edpt_xfer() since it schedule packet back-to-back here.